### PR TITLE
Round Number, Spawn Location, Tutorial, Controls

### PR DESCRIPTION
**Pull request by nisathnasar**

### CODE REVIEW

File: CopsAndRobbers/Assets/FindControls.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    /// @authors Nisath Mohamed Nasar
+    /// <summary>
+    /// Find appropriate Platform and display controls accordingly.
+    /// </summary>
+    public class FindControls : MonoBehaviour
+    {
+        public GameObject mobileControls, windowsControls;
+        
+#if UNITY_STANDALONE || UNITY_WEBPLAYER
+        private void EnableControls()
```
Comment (hsravat-4590) [inline CopsAndRobbers/Assets/FindControls.cs:17]: You could have all controls set to false and then only toggle one to true.
Also change name from windowsControls to desktopControls

**Review by hsravat-4590 — CHANGES_REQUESTED** [review]
See Comments

**Review by hsravat-4590 — CHANGES_REQUESTED** [review]

File: CopsAndRobbers/Assets/Scripts/PlayerScripts/PlayerHealth/PlayerRespawn.cs [hunk 2]
```diff
@@ -137,7 +137,7 @@ void ShowFloatingText()
         {
             Vector3 vector3 = new Vector3(transform.position.x, transform.position.y + 1.5f, transform.position.z  - 2f);
             var text = Instantiate(FloatingText, vector3, Quaternion.identity, transform);
-            text.GetComponent<TextMesh>().text = "Respawning";
+            text.GetComponent<TextMesh>().text = "reviving...";
```
[the remark this answers is not in the document]
Comment (nisathnasar) [reply CopsAndRobbers/Assets/Scripts/PlayerScripts/PlayerHealth/PlayerRespawn.cs:140]: I think smaller case is better here

File: CopsAndRobbers/Assets/FindControls.cs (hunk 1 above)
Comment (nisathnasar) [reply CopsAndRobbers/Assets/FindControls.cs:17]: fixed this can you recheck

**Review by hsravat-4590 — COMMENTED** [review]
Nice!

File: CopsAndRobbers/Assets/Scripts/PlayerScripts/PlayerHealth/PlayerRespawn.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (nisathnasar) [reply CopsAndRobbers/Assets/Scripts/PlayerScripts/PlayerHealth/PlayerRespawn.cs:140]: should I still change it

[the remark this answers is not in the document]
Comment (hsravat-4590) [reply CopsAndRobbers/Assets/Scripts/PlayerScripts/PlayerHealth/PlayerRespawn.cs:140]: alright then

**Review by hsravat-4590 — APPROVED** [review]

**Pull request merged**